src/instrumentation-node.ts

```diff
@@ -14,15 +14,27 @@
  * limitations under the License.
  */
 
-import { NodeSDK, resources } from '@opentelemetry/sdk-node';
+import { OTLPMetricExporter } from '@opentelemetry/exporter-metrics-otlp-http';
 import { HttpInstrumentation } from '@opentelemetry/instrumentation-http';
+import { metrics, NodeSDK, resources } from '@opentelemetry/sdk-node';
 import { ATTR_SERVICE_NAME } from '@opentelemetry/semantic-conventions';
 import { SERVICE_NAME } from './instrumentation';
 
+const metricExporter = new OTLPMetricExporter({
+  // TODO: Add the correct url address
+  // url: '',
+});
```

Comment:
Keep this empty, it should recognize `OTEL_EXPORTER_OTLP_ENDPOINT` env automatically.